playground/backend/internal/fs_tool/fs.go

```diff
@@ -117,5 +127,9 @@ func (l *LifeCycle) GetAbsoluteExecutableFilesFolderPath() string {
 
 // getFileName returns fileName by pipelineId and fileType ({pipelineId}.java for java SDK).
 func getFileName(pipelineId uuid.UUID, fileType string) string {
-	return fmt.Sprintf("%s.%s", pipelineId, fileType)
+	if fileType != "" {
+		return fmt.Sprintf("%s.%s", pipelineId, fileType)
+	} else {
+		return pipelineId.String()
```

Comment:
not very clear why there will be no extension here

Reply:
cause compiled go files have no extension and without this condition, the name of the binary file will be like "name."